src/components/molecules/ExecutionDetails/TestExecutionDetails/TestExecutionDetailsTabs.tsx

```diff
@@ -24,6 +27,7 @@ const LogOutput = lazy(() => import('@molecules').then(module => ({default: modu
 
 const TestExecutionDetailsTabs: React.FC = () => {
   const {data} = useContext(ExecutionDetailsContext);
+  const {pluginItems} = useContext(PluginsContext);
```

Comment:
We should rather have hooks for that, i.e.:

```ts
const usePluginDataList = <T[]>(name: string, defaults: T[] = []): T[] => {
  const {pluginItems} = useContext(PluginsContext);
  return pluginItems[name] || [];
};
```

This way it's safer, and i.e. such `usePluginDataList` hook could automatically handle the `insertToArray` and memoization